.github/workflows/testing.yml

```diff
@@ -47,7 +47,6 @@ jobs:
         cd build
         cmake \
           -DBUILD_SHARED_LIBS=ON \
-          -DDAKOTA_PYTHON=ON \
```

Comment:
other than it runs without it, do you have any evidence that it's unused?

Reply:
I don't, but @verveerpj removed this option a shortly before summer.